cupyx/cudnn.pyx

```diff
@@ -2482,7 +2482,7 @@ def fused_ops_execute(plan, var_pack):
 
 cpdef set_fused_ops_const_param_pack_attribute(
         Descriptor const_pack, int param_label, desc_or_scalar):
-    cdef int scaler
+    cdef int scaler  # no-cython-lint
```

Comment:
```suggestion
    cdef int scalar
```

Reply:
cython-lint doesn't pass otherwise. One solution it suggests is using underscore, but I believe that might become inconsistent with naming format.

Reply:
This was a typo (`scaler` -> `scalar`), it's used in the code below 😃